tex/latex/biblatex/biblatex.def

```diff
@@ -2803,6 +2804,29 @@
 
 \newbibmacro*{date}{\printdate}
 
+\DeclareFieldFormat{patentextradates}{\mkbibparens{#1}}
+\newbibmacro*{patentdate}{
+  \printdate%
```

Comment:
Here the `%` should not be needed.
```suggestion
  \printdate
```
One may want to think about using `\usebibmacro{date}%` here instead.
```suggestion
  \usebibmacro{date}%
```
Or maybe
```latex
  \newunit\newblock
  \usebibmacro{date}%
  \newunit\newblock
  \usebibmacro{patentdate}%
```
in `standard.bbx`.

In any case it would have to be made sure that this plays nicely with `mergedate` from `authoryear.bbx`.

Reply:
prefer this, note that maybe could legaldate ? Legislation act and other law have legal date:
law are voted, are enacted, then repelled. So maybe something more generic will help here

Reply:
I think I would prefer `patentdate` or maybe even `patentdates` over `legaldate` as the name of the bibmacro here. Even though laws and regulations (where `legaldate` is a good name) might be in a similar situation, the macro here is specifically targeting dates that have been named for patents only.